WooCommerce/Classes/Tools/ImageService/DefaultImageService.swift

```diff
@@ -1,6 +1,7 @@
 import Combine
 import Foundation
 import Kingfisher
+import UIKit
```

Comment:
Some import required tweaking after the move to SwiftPM. I haven't investigate, but my guess is that the packaging and linking is slightly different and therefore some headers that were implicitly part of the linkage no longer are. As you can see, the footprint is small.